src/Apps/ArtQuiz/ArtQuizInterface/maxHeight.ts

```diff
@@ -0,0 +1,34 @@
+import { scale } from "proportional-scale"
```

Comment:
For files like this, the folder idiom throughout Force is `ArtQuiz/Utils/` 

<img width="304" alt="Screen Shot 2022-09-19 at 4 54 18 PM" src="https://user-images.githubusercontent.com/236943/191138805-50f3cbb6-cc9b-48be-bbbe-484369e68b65.png">

Reply:
definitely! That's part of the plan for cleanup. I'm also going to break up my monolith file I'm working in. But, I'll make a note of it on my to-dos so I don't forget. (Because that's 100% classic me.)